packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -6,22 +6,25 @@ excerpt: >-
+* A non-OAuth 2.0 application (as is the case for a Classic org `/login/login.htm` endpoint), which uses the IP address and device identifier for rate-limiting.
 
-Client-based rate limiting for the `/login/login.htm` endpoint uses a combination of the user's IP address and Okta device identifier to provide even more granular, more targeted rate limits to a user, app, script, or server.
+The client access process can include requests to multiple API endpoints that differ based on your type of org: Okta Identity Engine or Okta Classic Engine.
 
-The [OAuth 2.0 `/authorize` endpoint](/docs/reference/api/oidc/#authorize) is the starting point for OpenID Connect flows such as the [Implicit flow](/docs/concepts/oauth-openid/#implicit-flow) or the [Authorization Code flow](/docs/concepts/oauth-openid/#authorization-code-flow). A request to this endpoint authenticates the user and either returns an authorization code or tokens to the client application as part of the callback response. The `/login/login.htm` endpoint launches the Okta-hosted sign-in widget, where users can enter their username and password to authenticate.
+The Classic Engine and the Identity Engine both include client-based rate-limiting for the OAuth 2.0 API endpoint `/authorize`. This endpoint uses a combination of the Client ID, user's IP address, and Okta device identifier to provide granular isolation for requests made to the OAuth 2.0 `/authorize` endpoint. This framework isolates rogue OAuth 2.0 clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations.
 
-Currently, the client-based rate limits apply to the `/login/login.htm` endpoint and OAuth 2.0 `/authorize` endpoint on both the Okta Org Authorization Server and any Custom Authorization Server. All Custom Authorization Servers share a [rate limit](/docs/reference/rate-limits/). The Org Authorization Server has a separate rate limit.
+The Identity Engine includes client-based rate limiting for multiple API entry points that implement the [Interaction Code grant type](/docs/concepts/interaction-code/), which performs a series of interactions and API calls between the user and authorization server.
+
+The Classic Engine also includes client-based rate limiting for the `/login/login.htm` endpoint, and uses a combination of the user's IP address and Okta device identifier to provide granular, targeted rate limits to a user, app, script, or server.
+
+The client-based rate limits for the OAuth2 `/authorize` endpoint apply to both the Okta Org Authorization Server and any Custom Authorization Server. All Custom Authorization Servers share a [rate limit](/docs/reference/rate-limits/). The Org Authorization Server has a separate rate limit.
```

Comment:
```suggestion
The client-based rate limits for the OAuth 2.0 `/authorize` endpoint apply to both the Okta Org Authorization Server and any Custom Authorization Server. All Custom Authorization Servers share the same [rate limit](/docs/reference/rate-limits/). The Org Authorization Server has a separate rate limit.
```
not sure if "a" should be replaced with "the same"? but just "a" sounds awkward

Reply:
Good catch